be/src/runtime/runtime_state.cpp

```diff
@@ -79,8 +79,6 @@ RuntimeState::RuntimeState(const TUniqueId& query_id, const TUniqueId& fragment_
           _num_rows_load_unselected(0),
           _num_print_error_rows(0) {
     _profile = std::make_shared<RuntimeProfile>("Fragment " + print_id(fragment_instance_id));
-    Status status = init(fragment_instance_id, query_options, query_globals, exec_env);
-    DCHECK(status.ok());
```

Comment:
How about passing `enable_pipeline_engine` to the constructor? then we needn't call `RuntimeState::init` outside. Otherwise, some constructors need call `init` and some needn't, which may be tricky and inconsistent.